src/openrct2/world/Climate.cpp

```diff
@@ -230,8 +230,21 @@ bool climate_is_raining()
 
 bool climate_is_snowing()
 {
-    if (gClimateCurrent.Weather == WEATHER_SNOW || gClimateCurrent.Weather == WEATHER_HEAVY_SNOW
-        || gClimateCurrent.Weather == WEATHER_BLIZZARD)
+    if (gClimateCurrent.Weather == WeatherType::Snow || gClimateCurrent.Weather == WeatherType::HeavySnow
+        || gClimateCurrent.Weather == WeatherType::Blizzard)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+bool climate_is_dry()
+{
+    if (gClimateCurrent.Weather == WeatherType::Sunny || gClimateCurrent.Weather == WeatherType::PartiallyCloudy
+        || gClimateCurrent.Weather == WeatherType::Cloudy)
```

Comment:
```suggestion
    return gClimateCurrent.Weather == WeatherType::Sunny || gClimateCurrent.Weather == WeatherType::PartiallyCloudy
        || gClimateCurrent.Weather == WeatherType::Cloudy;
```

This and all other `climate_is` functions can be a lot simpler by just returning directly.